### PR TITLE
Added Finnish translations

### PR DESCRIPTION
**Pull request by valstu**
Added Finnish 🇫🇮 translations to this project!

### CODE REVIEW

File: langs/fi.json [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    "CC": "australialainen / kookossaarelainen",
+    "CD": "kongolainen / (mahd. myös) kinshasankongolainen",
+    "CF": "keskiafrikkalainen",
+    "CG": "kongolainen / (mahd. myös) brazzavillenkongolainen",
+    "CH": "sveitsiläinen",
+    "CI": "norsunluurannikkolainen",
+    "CK": "uusiseelantilainen / cookinsaarelainen",
+    "CL": "chileläinen",
+    "CM": "kamerunilainen",
+    "CN": "kiinalainen  ",
```
Comment (sirtawast) [inline langs/fi.json:51]: Remove trailing spaces

**Review by sirtawast — APPROVED** [review]
Fellow Finn here, I have proofread the file and it looks good.

**Comment by aronhoyer** [comment]
don't forget to add your translations to the `data` object in `entry-node.js`, run tests and add it to the Supported languages section in `README.md` ([see contribution instructions here](https://github.com/sourcecode911/i18n-nationality#contribution))

**Comment by sourcecode911** [comment]
@aronhoyer i will do that in a different commit, seems to be that the author not responding.

**Pull request merged**